doc/act/zdw120_0_0.md

```diff
@@ -0,0 +1,270 @@
+---
+layout: documentation
+title: ZDW120 - ZWave
+---
+
+{% include base.html %}
+
+# ZDW120 Two Wire Wall Dimer
```

Comment:
Could you adapt your generation so that it makes sure to always have an empty line after every header? The current files will break the Jekyll renderer...